gems/pending/spec/appliance_console/database_replication_primary_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+require 'appliance_console/database_replication_primary'
+require 'linux_admin'
+require 'pg'
+
+NETWORK_INTERFACE = "eth0".freeze
```

Comment:
@carbonin Good catch. I put in the stub_const and forgot to remove the override. I'll remove the override here, on line 9.